guides/source/upgrading_ruby_on_rails.md

```diff
@@ -202,6 +202,11 @@ Rails 6.1. You are encouraged to enable `config.force_ssl` to enforce HTTPS
 connections throughout your application. If you need to exempt certain endpoints
 from redirection, you can use `config.ssl_options` to configure that behavior.
 
+### Redirection to HTTPS from HTTP will now use the 308 HTTP status code
```

Comment:
This is in the Rails 6.0 upgrade section, it should be moved up to the one for Rails 6.1.